src/scss/includes/components/panel.scss

```diff
@@ -52,6 +52,11 @@ $bottom-margin: 40;
   .panel {
     max-height: calc(100vh - (92px + 40px)); // 92px is panel.getBoundingClientRect().top
 
+    &.panel--pj {
+      max-height: calc(100vh - (92px + 40px + 36px));
```

Comment:
See my other comment about the position of the footer… At best you shouldn't have to change that.